src/stochastic_triple.jl

```diff
@@ -142,44 +163,55 @@ end
     stochastic_triple(X, p; backend=StochasticAD.PrunedFIs)
     stochastic_triple(p; backend=StochasticAD.PrunedFIs)
 
-If `p <: Real`, return the result of propagating the stochastic triple `p + ε` through the random function `X(p)`.
-If `p <: AbstractVector`, return a vector of stochastic triples of the same shape as `p`, containing the stochastic
-triples that result from perturbing the corresponding array elements of `p` one-by-one.
-When `X` is not provided, the identity function is used. The `backend` keyword argument describes the algorithm 
+For functor-like `p` (e.g. scalars, arrays), return an output of similar structure to p, containing
```

Comment:
I find it confusing to call arrays "functors",  the functor is the way a function defined on numbers is made a function defined on arrays. The dot in `a .+ b` is a functor for example.